public/app/features/explore/ContentOutline/ContentOutline.tsx

```diff
@@ -77,16 +99,21 @@ export function ContentOutline({ scroller, panelId }: { scroller: HTMLElement |
             aria-expanded={expanded}
           />
 
-          {outlineItems.map((item) => (
-            <ContentOutlineItemButton
-              key={item.id}
-              title={expanded ? item.title : undefined}
-              className={styles.buttonStyles}
-              icon={item.icon}
-              onClick={() => scrollIntoView(item.ref, item.title)}
-              tooltip={!expanded ? item.title : undefined}
-            />
-          ))}
+          {outlineItems.map((item) => {
+            return (
+              <ContentOutlineItemButton
+                key={item.id}
+                title={expanded ? item.title : undefined}
+                className={styles.buttonStyles}
+                icon={item.icon}
+                onClick={() => {
+                  scrollIntoView(item.ref, item.title);
+                }}
```

Comment:
Nit: It doesn't change the behavior, just adds extra `{ ... }` which is not needed.